app/views/spree/checkout/payment/_gateway.html.haml

```diff
@@ -1,4 +1,11 @@
 .row
```

Comment:
I think this will have to be in the _stripe_connect.html.haml partial for now, we have other gateways (PayPal, Pin) for which this will not apply. Would be good to only show this field if there are actually some savedCreditCards.